packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
-# Forms
+# Richtlijnen voor formulieren
 
-For individual form component guidelines please consult the documentation for that specific component.
+_Status: **In ontwikkeling**_
 
-_Status: **In development**_
+_Individuele formulier elementen en hun specifieke richtlijnen zijn ondergebracht in de componentenbibliotheek._
 
-## Introduction
+Met een formulier nodig je iemand uit tot een dialoog met jouw organisatie. Het is niet slechts een manier om jouw database te vullen maar een onderdeel van de gehele reis die iemand door jouw product maakt. Ontwerp een formulier vanuit degene die deze in moet vullen.
 
-A form is the moment the user enters into a dialogue with our products and organisation. It should therefore be as clear and convenient as possible.
+Iemand komt voor wat er zich aan de andere kant van het formulier bevindt; een antwoord op een vraag, een besteld product, een ingelogde staat, etc. Een formulier is een barrière tussen hen en dit doel. Het moet zo duidelijk mogelijk zijn hoe iemand dit doel kan bereiken en zo makkelijk mogelijk om te voltooien.
```

Comment:
Ik zou het woord "formulier" denk ik eerder in de eerste zin verwachten om het nog meer to the point te maken? En ik merk dat "Iemand komt om..." heel erg als een scenario lees, terwijl het hier meer een algemeen punt is. Misschien zou iets als "Iedereen komt bij een formulier voor wat zich aan de andere kant bevindt..." het duidelijker maken?

Reply:
Ik neem deze openingsalinea’s opnieuw onder de loep.